test/libsolidity/SolidityOptimizer.cpp

```diff
@@ -485,7 +485,7 @@ BOOST_AUTO_TEST_CASE(constant_optimization_early_exit)
 	auto start = std::chrono::steady_clock::now();
 	compileBothVersions(sourceCode);
 	double duration = std::chrono::duration<double>(std::chrono::steady_clock::now() - start).count();
-	BOOST_CHECK_MESSAGE(duration < 20, "Compilation of constants took longer than 20 seconds.");
+	BOOST_CHECK_MESSAGE(duration < 60, "Compilation of constants took longer than 60 seconds.");
```

Comment:
Maybe a better solution would be to not run this test if ASan is activated.

Reply:
Created a new issue for this #7191